libr/core/cmd_meta.c

```diff
@@ -905,9 +905,8 @@ void r_comment_vars(RCore *core, const char *input) {
 		eprintf ("Can't find function here\n");
 		return;
 	}
-	oname = name = strdup (input + 2);
-	while (*name == ' ') {
-		name++;
+	if (input[1]) {
+		name = r_str_trim_dup (input[1] != ' ' ? input + 2 : input + 1);
```

Comment:
this condition seems wrong/unnecessary, checking for a space when passing the string to a function that trim spaces is unnecessary